CONTRIBUTING.md

```diff
@@ -3,25 +3,29 @@ Contributing to Ansible-lint
 
```

Comment:
@gundalow AFAIR this file can also live under `.github/` or even under `docs/`. If this project is going to have RTD it'll be good to convert this to RST (probably not now).

Reply:
Trouble with RST is that github doesn't render it well (iirc). CONTRIBUTING.md and README.md are key resources that github make highly visible to contributors.

I would suggest adding a md to rst conversion as part of the RTD docs generation if you want the docs in RST format without redundancy